internal/controller/kyma/controller.go

```diff
@@ -264,6 +298,14 @@ func (r *Reconciler) syncStatusToRemote(ctx context.Context, kcpKyma *v1beta2.Ky
 	return nil
 }
 
+// ValidateKymaSpec validates the Kyma spec.
+func (r *Reconciler) ValidateKymaSpec(kyma *v1beta2.Kyma) error {
```

Comment:
please consider rename it to indicate this function only validate global channel, but `ValidateKymaSpec` is too generic.

Reply:
renamed to `ValidateDefaultChannel`